content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/improve-how-you-give-feedback-to-primary-pupils.md

```diff
@@ -0,0 +1,68 @@
+---
+
+# Improve how you give feedback to primary pupils
+
+<strong class="govuk-tag">Example</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** Kensington Primary School
+
+**Location:** London
+
+**Phase:** Primary
+
+**Number of pupils:** 700
+
+**Contact details:** Email the school office at <info@kensington.ttlt.academy> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Our approach has helped to reduce workload by empowering teachers to make professional judgements about when and how to give feedback to students. This means that all the time and effort of marking and feedback is focused on having an impact on the students rather than ticking a box. 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ben Levinson, Headteacher
+
+{inset-text}
+
+We interrogated the most recent research on effective feedback and discussed how it could work in our own practice with our whole staff. We developed our approach based on the question “why are we giving this feedback?”  
+
+As a school we went through a number of iterations of the approach, which allowed our staff to gradually change their practice rather than having to transform overnight. This helped us prevent added workload that often comes with sudden big changes. Throughout the changes we gathered feedback and provided support for staff who needed help and guidance. 
+
+{/inset-text}
+
+## Why give feedback 
+
+There are two main reasons we give feedback. Firstly, to motivate the child. Secondly, we can see an opportunity for learning by: 
+
+* addressing a misunderstanding 
+
+* reinforcing a skill or key piece of information 
+
+* extending a child’s understanding or ability to do something 
+
+## When to give feedback 
+
+When and how feedback is given matters. It needs to be based on what will most positively impact that child's progress. As a school, we encourage teachers to use their professional judgment to decide when it is the right time to provide verbal feedback, to surface mark, highlighting, or when a next step comment is the right approach. 
+
+## Support staff to make decisions 
+
+Whilst this level of professional decision making is our ultimate goal, we understand that these judgments are difficult and, often, it can take time to develop a real understanding of when to use a particular type of feedback. If teachers are ever unsure, they can speak to another member of staff, Phase Leader or member of SLT, any of whom will be more than happy to offer advice and support. If there are queries, SLT will have professional dialogues with teachers to understand why they have made certain choices and may offer further support and guidance. 
```

Comment:
Some of these sentences are quite a bit longer than 25 words - can they be broken down a little bit?:

'Whilst this level of professional decision making is our ultimate goal, we understand that these judgements are difficult. Often, it can take time to develop a real understanding of when to use a particular type of feedback. If teachers are ever unsure, they can speak to another staff member, phase leader or SLT member, who can offer further advice and support. If there are queries, SLT will have conversations with teachers to understand why they have made certain choices and offer further support and guidance.